js/templates/chatApp.html

```diff
@@ -0,0 +1,25 @@
+<div class="newChatButton">
+    <a class="btn btn-newChat js-newChat">+ <span class="hide">New</span></a>
+    <div class="btn btn-closeChat js-closeChat ion-close-round"></div>
```

Comment:
For js-classes on DOM items, I've been (mostly) following the convention of putting the view name first, so in this case "js-chatNew" and "js-chatClose".

Not an issue, but we may want to be consistent..